src/protocols/ethereum/subgraph.js

```diff
@@ -12,7 +12,6 @@ module.exports = class EthereumSubgraph {
 
   validateManifest() {
     return this.validateAbis()
-      .concat(this.validateContractAddresses())
```

Comment:
It has not been removed, it has been moved to the "upper scope".

The `Subgraph` class calls `validateManifest` for the `protocol` it needs to validate, now instead of each protocol having to implement this, it's decided on an upper scope, [here](https://github.com/graphprotocol/graph-cli/pull/783/commits/9a77d795e93f5164d9b4aa35bab2b867ad6f2925#diff-c062062b77741126d1ae10b39c359caa60e8b08315c3f79f129eed2f7fa0fb28R148).